src/mango_cursor_text.erl

```diff
@@ -286,13 +283,18 @@ update_query_args(CAcc) ->
 
 get_limit(CAcc) ->
     Total = CAcc#cacc.limit + CAcc#cacc.skip,
```

Comment:
change function to `erlang:min(get_dreyfus_limit(), CAcc#cacc.limit + CAcc#cacc.skip).`